docs/installation/linux/fedora.md

```diff
@@ -12,80 +12,96 @@ weight=-3
 
 # Fedora
 
-Docker is supported on Fedora version 22, 23, and 24. This page instructs you to install
-using Docker-managed release packages and installation mechanisms. Using these
-packages ensures you get the latest release of Docker. If you wish to install
-using Fedora-managed packages, consult your Fedora release documentation for
-information on Fedora's Docker support.
+Docker is supported on Fedora version 22, 23, and 24. This page instructs you to
```

Comment:
Consider "Fedora 22 and higher". s/This page instructs you to install /These instructions install Docker using release packages and installation mechanisms managed by Docker, to be sure that you get the latest version of Docker.